### PR TITLE
Rough outline of how it works part

### PR DESCRIPTION
**Pull request by shikolay**

### CODE REVIEW

**Comment by logantheproblemsolver** [comment]
I really like this version! I like how specific it is in the how it works page!

**Review by malcolmkiano — COMMENTED** [review]
I think this is acceptable. Great wording and description

File: covidwebapp/scandetector/templates/how-it-works.html [hunk 1]
```diff
@@ -28,13 +28,43 @@
...
-        Our model is an ensemble of many pre-trained deep COVNets. Networks that were pre-trained on thorax-image classification tasks were transferred to COVID-19 detection. Then we used a mixture model for aggregation of multiple pre-trained weak classifiers.
+- Intro
+  - Goal of this project was to make advancements in Machine Learning helps fight COVID 19 pandemic by making Deep Learning Neural Networks easy to use for everyone.
+- Details about algorithm
+  - The whole system essentially consist of 2 parts.
+    - Web Application
+      - Web application is what are using right now. It allows to upload scans and run prediction models.
+    - Machine learning part
+      - Our algorithm consist of 4 phases each with specific detection in mind
+        - First phase try to predict probability of the file being xray scan at all. We develop a CNN (convolutional neural network) that we trained on set of images that was mix of xray scans and just regular photos or images of objects .
```
Comment (nastikas) [inline covidwebapp/scandetector/templates/how-it-works.html:39]: The first part - identify if the uploaded image is Xray or not.

Comment (nastikas) [reply covidwebapp/scandetector/templates/how-it-works.html:39]: We develop a CNN (convolutional neural network) that we trained on a set of images that was a mix of Xray scans and just regular photos or images of objects.

File: covidwebapp/scandetector/templates/how-it-works.html [hunk 2]
```diff
@@ -28,13 +28,43 @@
...
+        - Second phase is focusing on predicting probability of file beign xray of healthy lungs. This phase use DenseNet201 trained on CheXpert dataset where labels were reduced to binary outcome (healthy/unhealthy)
+        - Third phase similar to previous one, with exception that focus was on detection pneumonia and everything else. Same architecture (DenseNet201) and dataset was used(ChexPert)
```
Comment (nastikas) [inline covidwebapp/scandetector/templates/how-it-works.html:41]: The third phase is similar to the previous one

**Pull request merged**